### DISCUSSION TITLE
FORCE_REDOWNLOAD= false is not working

### DISCUSSION THREAD

**Discussion by Bnewn** [itzg/docker-minecraft-server#2526] [Q&A]
Every time I restart the docker container, it tries to download another jar image from spigot. Is there a way to stop this and just start the server when the container is started? I have tried FORCE_REDOWNLOAD= false as stated in the documentation but still has no effect! Please help!

**Comment by itzg** [comment]
Bukkit provides zero support for binary downloads and the container setup has to use a third party download site with no support for automation. So, I'm not able nor willing to provide any further enhancements for Bukkit/Spigot retrieval. 

I would recommend switching to PaperMC which is plugin compatible and is much better supported.
